libdevcore/Worker.h

```diff
@@ -29,74 +29,60 @@
 namespace dev
 {
 
-enum class IfRunning
-{
-	Fail,
-	Join,
-	Detach
-};
-
 enum class WorkerState
 {
-	Starting,
-	Started,
-	Stopping,
-	Stopped,
```

Comment:
`Starting` and `Killing` is now `m_requestedState == WorkerState::Started` / `WorkerState::Killing`, since "Starting" and especially "Killing" is not really the state of the worker (see description of this PR).
Perhaps it is better to view this PR not in the diff but in the full view. Note the absence of any of the special functions of `std::atomic`.